## 7- A frequência de "Movies" de todos os dados e comparativamente em relação as duas plataformas

In [1]:
# importamos a biblioteca pandas
import pandas as pd

### Considerando TODOS os dados

In [2]:
# carregamos dados do Netflix e Amazon
netflix = pd.read_csv('netflix_titles.csv')
amazon_prime = pd.read_csv('amazon_prime_titles.csv')
# concatenamos os dados
all_data = pd.concat([netflix, amazon_prime], ignore_index=True)
# filtramos apenas por filmes
all_data = all_data[all_data['type']=='Movie']
# resetamos os indices
all_data.reset_index(inplace=True, drop=True)
# calculamos o numero de registros (frequencia) do tipo TV Show
len(all_data)

13945

### Dados do Netflix

In [3]:
# carregamos dados do Netflix
netflix = pd.read_csv('netflix_titles.csv')
# filtramos apenas por TV Show
netflix = netflix[netflix['type']=='Movie']
# resetamos os indices
netflix.reset_index(inplace=True, drop=True)
# calculamos o numero de registros (frequencia) do tipo TV Show
len(netflix)

6131

### Dados da Amazon

In [4]:
# carregamos dados do Netflix
amazon = pd.read_csv('amazon_prime_titles.csv')
# filtramos apenas por TV Show
amazon = amazon[amazon['type']=='Movie']
# resetamos os indices
amazon.reset_index(inplace=True, drop=True)
# calculamos o numero de registros (frequencia) do tipo TV Show
len(amazon)

7814

## Análise Comparativa
#### Na plataforma Netflix tem-se 6131 registros o que representa  (6131 / 13945) % ou 44% do total de registros do tipo Movie.
#### Na plataforma Amazon 7814 registros, equivalente a (7814 / 13945) % ou  56% do total de registros do tipo Movie.
#### Existe uma diferença de 12% entre plataformas. On a plataforma Amazon possui a maior porcentagem de Movies.